.coveragerc

```diff
@@ -1,4 +1,11 @@
 # .coveragerc to control coverage.py
+
+# The files in the scripts folder will be deleted once the migration to Docker is complete.
```

Comment:
We are probably never migrating to Docker. I think you can drop this paragraph entirely.